donkeycarManagerFront/src/components/carInfo.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="carCard">
+    <va-card stripe :stripe-color=car.color>
+        <va-card-title><b>{{ car.name }}</b></va-card-title>
+        <va-card-content class="content-card ">
+          <b>Ip :</b> {{car.ip}} <br>
+          <b> Status:</b>
+          <va-badge v-if="car.current_stage === 'DRIVE'" text="🎮 Drive" color="warning" class="mr-4" />
+          <va-badge v-if="car.current_stage === 'RECORD'" text="🎥 recording data" color="#8338ec" class="mr-4" />
+          <va-badge v-if="car.current_stage === 'AI_ASSISTED'" text="🧪 AI assisted" color="#06d6a0" class="mr-4" />
+          <va-badge v-if="car.current_stage === 'MAINTENANCE'" text="🧰 Maintenance" color="fe5f55" class="mr-4" />
+          <va-badge v-if="car.current_stage === null" text="❌ no status" color="#fe5f55" class="mr-4" />
```

Comment:
can be simplified to v-if="!car.current_stage"

Reply:
oh, my bad I have made a bull shit